src/services/orders/order.service.ts

```diff
@@ -0,0 +1,30 @@
+import { BASE_URL } from "../../constants/url.constant"
+import { CreateOrderDTO } from "../../interfaces/orders/order.interface";
+
+const ORDERS_BASE_URL = `${BASE_URL}/orders`
+
+const orderService = {
+  create: async (orderData: CreateOrderDTO) => {
+    try {
+      const response = await fetch(ORDERS_BASE_URL, {
```

Comment:
This can be used from the constants file as I mentioned. It will be something like this: `API_BASE_URL + API_URL.ORDERS`